Okta.AspNet/Okta.AspNet.csproj

```diff
@@ -3,9 +3,9 @@
   <PropertyGroup>
     <Description>Official Okta middleware for ASP.NET 4.5+. Easily add authentication and authorization to ASP.NET applications.</Description>
     <Copyright>(c) 2019 Okta, Inc.</Copyright>
-    <Version>2.0.0</Version>
+    <Version>2.0.1</Version>
     <Authors>Okta, Inc.</Authors>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
We should talk about this change since it will drop support for older .NET versions our customers are using and it's def a major change. We could potentially multitarget diff .NET frameworks and use specific dependency versions according to the target version.  See [this ](https://github.com/okta/okta-aspnet/commit/75af987015da3a8ad78cbce2c8b8d738ce9ae49d#diff-3f074fa393aff101aa6e63411d18a67e60a23acf006ffe065748d68c89b796c0R32) example.

Reply:
There is a security issue with the `IdentityModel` library, the updated version of it supports 4.6.1 and 4.7.2.  We can multi target those if you think it makes sense.

Reply:
Out of curiosity I checked Microsoft's .Net Framework support policy (https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-framework#sha1-retirement) and it looks like versions prior to 4.6.2 are no longer supported as of yesterday 04/26/2022.  Considering that, I think this target framework change should be safe.